app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str):
+    cmd_list = command.split()
+    cmd = cmd_list[0]
+    old_file = cmd_list[1]
+    new_file = cmd_list[2]
+
+    if old_file == new_file or cmd != "cp":
+        return print("Something wrong!")
+    else:
+        with open(old_file, "r") as file, open(new_file, "w") as new_file:
```

Comment:
`file` and `new_file` aren't really meaningful, better name it like `source_file` and `destination_file`, or smth like that